src/deepsparse/server/cli.py

```diff
@@ -90,9 +90,20 @@ def main():
     default="info",
     help="Sets the logging level.",
 )
-def config(config_path: str, host: str, port: int, log_level: str):
+@click.option(
+    "-r",
+    "--hot-reload-config",
+    is_flag=True,
+    default=False,
+    help="Hot reload the config whenever the file is updated.",
```

Comment:
prefer to be a little more descriptive here, ie saying deployed routes will be updated based on latest config

Reply:
Updated